library/gen_x12_837.inc.php

```diff
@@ -89,35 +89,9 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim = false)
     ++$edicount;
 
     if ($claim->federalIdType() == "SY") { // check entity type for NM*102 1 == person, 2 == non-person entity
-        $tempName = $claim->billingFacilityName();
```

Comment:
looked at that class. definitely has good potential to modernize to psr4. would be glad to give pointers as needed (for example, in that file, migrate the 2 function at the top to be private functions within the class).